avro-serializer/src/main/java/io/confluent/kafka/serializers/KafkaAvroSerializerConfig.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.kafka.serializers;
+
+import io.confluent.common.config.ConfigDef;
+
+import java.util.Map;
+
+/**
```

Comment:
I'm guessing this was auto-added by your IDE - our convention is not to have this attribution comment.